src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+
+    public double getRandomLeg() {
```

Comment:
```suggestion
    private double getRandomValue() {
```
this is helper method, make it private and move below public methods